From: Rob Evers <revers@redhat.com>
Date: Thu, 28 Oct 2010 17:53:54 -0400
Subject: [scsi] lpfc: update version for 8.2.0.86 driver release
Message-id: <1288288434-4069-4-git-send-email-revers@redhat.com>
Patchwork-id: 28970
O-Subject: [RHEL5.6 PATCH 3/3] lpfc: Update lpfc version for 8.2.0.86 driver
	release
Bugzilla: 645881

From: Rob Evers on behalf of Emulex <revers@redhat.com>


diff --git a/drivers/scsi/lpfc/lpfc_version.h b/drivers/scsi/lpfc/lpfc_version.h
index ecefe0c..82ae656 100644
--- a/drivers/scsi/lpfc/lpfc_version.h
+++ b/drivers/scsi/lpfc/lpfc_version.h
@@ -18,7 +18,7 @@
  * included with this package.                                     *
  *******************************************************************/
 
-#define LPFC_DRIVER_VERSION "8.2.0.85"
+#define LPFC_DRIVER_VERSION "8.2.0.86"
 
 #define LPFC_DRIVER_NAME		"lpfc"
 #define LPFC_SP_DRIVER_HANDLER_NAME	"lpfc:sp"
